### ISSUE TITLE
Vivaldi

### ISSUE THREAD

**Issue by ubuntudroid** [mandrigin/AlfredSwitchWindows#28]
Any plans to support Vivaldi tabs in the future?

**Comment by mandrigin** [comment]
@ubuntudroid yeah, but last time I checked they (and Firefox) lacked APIs bindings for AppleScript that this workflow requires.

**Comment by ubuntudroid** [comment]
Ah, unfortunate, but thanks for keeping it on the radar! 🙏
